pkg/resources/delete_cluster_gce.go

```diff
@@ -400,7 +400,7 @@ func (d *clusterDiscoveryGCE) listGCEDisks() ([]*ResourceTracker, error) {
 		}
 
 		for _, u := range t.Users {
-			tracker.blocked = append(tracker.blocked, typeInstance+":"+t.Zone+"/"+gce.LastComponent(u))
+			tracker.blocked = append(tracker.blocked, typeInstance+":"+gce.LastComponent(t.Zone)+"/"+gce.LastComponent(u))
```

Comment:
Related to a change that was added [here](https://github.com/kubernetes/kops/pull/2262), I had not realized `t.Zone` was referencing the URL.